contents/articles/2017-01_image_user_auth/index.md

```diff
@@ -0,0 +1,25 @@
+--- 
+title: Personalized Image-based User Authentication using Wearable Cameras
+date: Wed Jan 11 2017 11:18:18 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Ngu Nguyen et al., propose a novel personalized user authentication mechanism that generates an image-based passwords from egocentric videos captured by wearable cameras..."
+featured_img: "../../../../media/images/blog/user_auth.png"
+featured_img_thumb: "../../../../media/images/blog/user_auth.png"
+---
+
+Personal devices are a part of our everyday life that operate and store users private data. Such devices face an array of threats that have become a primary concern for users. The existing authentication mechanism requires users to register and remember one static password for each device at a time. However static passwords are vulnerable to various methods of attacks. 
+
+<img src="../../../../media/images/blog/user_auth.png" class='Feature-image' alt="Image-Based Authentication Challenges Generated From Egocentric Videos">
+
+<div class="small u-padBottom--2">Image Source: Fig 1. Page 2. - [Personalized Image-based User Authentication using Wearable Cameras](https://arxiv.org/pdf/1612.06209.pdf)</div>gul
+
+[Ngu Nguyen](http://ambientintelligence.aalto.fi/team.html) et al., propose a novel personalized user authentication mechanism that generates an image-based passwords from egocentric videos captured by wearable cameras. 
+
+One of the process of generating authentication challenges uses Pupil Lab's [Pupil headset](https://pupil-labs.com/store/#pupil) to capture egocentric videos and gaze data. [Pupil eye tracker](https://pupil-labs.com/store/#pupil) was used to track eye fixations to reveal user's attention which are demonstrated to be useful in [video summarization](http://www.cv-foundation.org/openaccess/content_cvpr_2015/papers/Xu_Gaze-Enabled_Egocentric_Video_2015_CVPR_paper.pdf) to provide information of wearer's visual behavior.
```

Comment:
The researchers use Pupil to capture egocentric video and gaze data. Fixations from gaze data can be used to discriminate potentially meaningful or salient segments of the egocentric video. These segments can then be used to split up the video into groups of personalized images used in the auth challenge.